commands/apps.go

```diff
@@ -0,0 +1,393 @@
+/*
+Copyright 2018 The Doctl Authors All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package commands
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/digitalocean/doctl"
+	"github.com/digitalocean/doctl/commands/displayers"
+	"github.com/digitalocean/godo"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+)
+
+// Apps creates the apps command.
+func Apps() *Command {
+	cmd := &Command{
+		Command: &cobra.Command{
+			Use:     "apps",
+			Aliases: []string{"a"},
+			Short:   "[Beta] Display commands for working with apps",
+			Long:    "[Beta] The subcommands of `doctl app` manage your apps.",
```

Comment:
```suggestion
			Long:    "[Beta] The subcommands of `doctl app` manage your apps.",
			Hidden:  true,
```

This would hide it from the main command list, but calling `doctl apps -h` would still show the sub-commands.